site/docs/v1/tech/tutorials/gating/_troubleshooting-gating.adoc

```diff
@@ -0,0 +1,32 @@
+=== Links Emailed to Users Don't Function As Expected
+
+
+Users cannot verify their application registration or verify their email. The link or code does not work (it is expired or there is some other error).
+
+====  Check the Email Template
+
+Ensure you have selected the correct email template.
+
+**For email verification**
+
+If attempting to verify email, ensure you have selected the appropriate template (`email verification` as opposed to `registration verification` in this instance)
+
+image::tutorials/email-verify-right.png[Correct Template Email Verification,width=1200]
+image::tutorials/email-verify-wrong.png[Incorrect Template Email Verification,width=1200]
```

Comment:
I like the 'checkmark' and 'prohibited' stamps, but the prohibited one should be red.